rest/server_context.go

```diff
@@ -411,12 +411,12 @@ func (sc *ServerContext) _getOrAddDatabaseFromConfig(config *DbConfig, useExisti
 
 	bucket, err := db.ConnectToBucket(spec, func(bucket string, err error) {
 
-		msg := fmt.Sprintf("%v dropped Mutation feed (TAP/DCP) due to error: %v, taking offline", bucket, err)
-		base.Warn(msg)
+		msgFormatStr := "%v dropped Mutation feed (TAP/DCP) due to error: %v, taking offline"
+		base.WarnR(fmt.Sprintf(msgFormatStr, base.UD(bucket), err))
```

Comment:
Need to drop the sprintf, the log functions already do this.

Reply:
Fixed